extensions/objects/templates/sanitized-url-editor.component.ts

```diff
@@ -46,15 +51,21 @@ export class SanitizedUrlEditorComponent {
   @Output() valueChanged = new EventEmitter();
   schema: SanitizedUrlSchema = {
     type: 'unicode',
-    validators: [{
-      id: 'is_nonempty'
-    }],
+    validators: [
+      {
+        id: 'is_nonempty'
+      },
+      {
+        id: 'is_regex_matched',
+        regexPattern: '(^https:\\/\\/.*)|(^(?!.*:\\/\\/)(.*))'
```

Comment:
Should the "s" in "https" be optional?

Reply:
In the issue description it was that only 'https' should be accepted but I'm not 100% sure.

Reply:
We're going to only allow "https" from now on. So I think the regex looks right.